alpha/apps/kaltura/lib/renderers/kRendererPdfFile.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once(dirname(__file__) . '/kRendererBase.php');
+
+/**
+ * Created by IntelliJ IDEA.
+ * User: Tali.Harash
+ * Date: 9/7/2015
+ * Time: 2:54 PM
+ */
+class kRendererPdfFile implements kRendererBase
+{
+
+
+    private $pdfObj;
+
+    public function __construct(PdfGenerator $obj)
+    //public function __construct($obj)
+    {
+        $this->pdfObj = $obj;
+    }
+
+    public function validate()
+    {
+        return true;
+    }
+
+    public function output()
+    {
+        try
+        {
+            KalturaLog::debug("PDF::: kQuizPdf output");
+            echo $this->pdfObj->Submit();
+            //return $this->pdfObj->Submit();
+            //echo($this->pdfObj);
+
+            KExternalErrors::dieGracefully();
+        }
+        catch (KalturaAPIException $e)

```

Comment:
1. What might cause this exception? 
2. This is an API exception and shouldn't be used here.
3. If exception is expected, log it as error and not as debug message.